hivemind/utils/__init__.py

```diff
@@ -4,7 +4,14 @@
 from hivemind.utils.logging import get_logger, use_hivemind_log_handler
 from hivemind.utils.mpfuture import *
 from hivemind.utils.nested import *
-from hivemind.utils.networking import *
+from hivemind.utils.networking import (
+    Endpoint,
+    choose_ip_address,
+    get_free_port,
+    get_port,
+    log_visible_maddrs,
+    replace_port,
+)
```

Comment:
Just as a reminder, the goal here is to incrementally move from wildcard imports to explicit enumeration of all exposed functions and classes. All public objects remain available for direct imports, but the higher we go in the import hierarchy, the less we want to expose (mostly the key functionality at the top level of hivemind, which also helps on autocompletion)

Reply:
Kept two imports: the one that is used externally (`log_visible_maddrs`) and the one that is used in a couple of tests (`get_free_port`), since I want to restrict the scope of this PR. In follow-up PRs, we can actually move `get_free_port` to `test_utils`, since it is not used in any of hivemind-internal code